scylla/src/transport/errors.rs

```diff
@@ -734,6 +733,12 @@ impl BrokenConnectionError {
     }
 }
 
+impl std::fmt::Display for BrokenConnectionError {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(f, "Connection broken, reason: {}", self.0)
+    }
+}
+
```

Comment:
Hmm, why this change from auto-generated Display to boilerplate'ish Display impl? Why not the other way round, so that all three error types use `#[error()]` shorthand?

Reply:
True.